lang.go

```diff
@@ -115,6 +116,8 @@ func CodeToLang(code string) Lang {
 		return Ces
 	case "cmn":
 		return Cmn
+	case "yue":
```

Comment:
The languages are in order alphabetically (in English). These lines should go above `case "zul":`.